public/favicon/site.webmanifest

```
@@ -0,0 +1 @@
+{"name":"Home | Design and Code","short_name":"DC","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
{"name":"Home | Design and Code","short_name":"DC","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}

```

---

Done ✅